fudy/MenuBar.qml

```diff
@@ -27,42 +27,50 @@ Rectangle {
 		}
 	}
 
-	Row {
+	RowLayout {
 		id: toolIcons
-
-		spacing: 23
+		width: 295
+		height: 30
 		anchors {
-			verticalCenter: root.verticalCenter
-			right: root.right
+			verticalCenter: parent.verticalCenter
+			right: parent.right
 		}
+
+		spacing: 23
 		Image {
-			width: 30
-			height: 30
+			Layout.preferredWidth: 30
+			Layout.preferredHeight: 30
+			fillMode: Image.PreserveAspectCrop
 			source: "qrc:/image/home.svg"
 		}
 		Image {
-			width: 30
-			height: 30
+			Layout.preferredWidth: 30
+			Layout.preferredHeight: 30
+			fillMode: Image.PreserveAspectCrop
 			source: "qrc:/image/search.svg"
 		}
 		Image {
-			width: 30
-			height: 30
+			Layout.preferredWidth: 30
+			Layout.preferredHeight: 30
+			fillMode: Image.PreserveAspectCrop
 			source: "qrc:/image/plus.svg"
 		}
 		Image {
-			width: 30
-			height: 30
+			Layout.preferredWidth: 30
+			Layout.preferredHeight: 30
+			fillMode: Image.PreserveAspectCrop
 			source: "qrc:/image/bell.svg"
 		}
 		Image {
-			width: 30
-			height: 30
+			Layout.preferredWidth: 30
+			Layout.preferredHeight: 30
+			fillMode: Image.PreserveAspectCrop
 			source: "qrc:/image/settings.svg"
 		}
 		Image {
```

Comment:
Please separate this Image to the new Component (maybe in the future you will create custom MenuBarItem)

Reply:
```
\\ MenuBarItem.qml
Item {
        id: root
        property string sourceUrl: ""
        Layout.preferredWidth: 30
	Layout.preferredHeight: 30
        Image {
			        fillMode: Image.PreserveAspectCrop
			        source: root.sourceUrl
        }
}
\\ other.qml
MenuBarItem {
     sourceUrl: "qrc:/image/settings.svg"
}
MenuBarItem {
     sourceUrl: "qrc:/image/home.svg"
}
```

Reply:
got it, thank you